libmscore/measurebase.cpp

```diff
@@ -307,8 +307,11 @@ void MeasureBase::layout()
 
 void MeasureBase::triggerLayout() const
       {
-      if (prev() || next()) // avoid triggering layout before getting added to a score
-            score()->setLayout(tick(), -1, this);
+      // for measurebases within other measurebases (e.g., hbox within vbox), use parent
+      const MeasureBase* mb = parent() && parent()->isMeasureBase() ? toMeasureBase(parent()) : this;
```

Comment:
Please add a `const` qualifier and parentheses:

    const MeasureBase* const mb = (parent() && parent()->isMeasureBase()) ? toMeasureBase(parent()) : this;